lib/absinthe/phase/document/arguments/flag_invalid.ex

```diff
@@ -18,6 +18,10 @@ defmodule Absinthe.Phase.Document.Arguments.FlagInvalid do
     {:ok, result}
   end
 
+  defp handle_node(%{schema_node: nil, flags: %{scalar_child: _}} = node) do
```

Comment:
Need to figure out a way to flag the children to indicate we don't want to flag these nodes as invalid

Reply:
went with a different approach